README.md

```diff
@@ -25,14 +25,50 @@ The service is composed by the following components:
 - *fathom-proxy*: the [nginx](https://www.nginx.com) service used as reverse-proxy
 
 ## Deployment / Configuration
-The templates provided in the `openshift` folder include everything that is necessary to create the required builds and deployments.
+The templates provided in the `openshift` folder include everything that is necessary to create the required builds and deployments.  
 
-Since there are interdependencies between deployment configurations, please make sure to follow this order when creating them for the first time:
-1) build and deploy the database
-2) build and deploy the Fathom analytics server
-3) build and deploy the proxy
+To run Fathom on openshift you **MUST** install [openshift-developer-tools](https://github.com/BCDevOps/openshift-developer-tools) and have them available on your path  
+
+By default, Fathom uses the artifactory docker registry. If you are going to keep the default settings artifactory **MUST** be enabled in your OCP cluster, otherwise you will have to tweak the param file to specify your docker registry.  
+
+
+#####Running with Artifactory:
+There should already be a "artifacts-default-\*\*\*\*\*\*" secret in the tools environment of your openshift cluster. Copy the username and password of this 
+secret and run the following command in each environment you wish to build/deploy to. (example: tools and dev / tools and prod)
+~~~
+oc create secret docker-registry artifactory-creds \
+    --docker-server=docker-remote.artifacts.developer.gov.bc.ca \
+    --docker-username=<our username from secret> \
```

Comment:
I'd recommend reading the credentials from the secret, rather than copy/paste.  Example of how to read a secret can be found here; https://github.com/BCDevOps/openshift-developer-tools/blob/master/bin/ocFunctions.inc#L1788.  Or integrate the feature into the `manage` script that is already extending those tools.  For bonus marks you could implement the script as a function in the `ocFunctions.inc` file so it can be reused.

Reply:
We don't know what the ending of the artifactory secret will be. A string is created randomly and attached to the end of artifactory-default-. We might have to get a list of secrets and select one that starts with artifactory-default. Another issue is that we need a way for the user to use artifactory creds, docker creds, etc. how will we do this from the manage script?

Reply:
maybe we could have a SECRET_NAME in the param file to specify what type of secret to use?

Reply:
I still think this part should be linked to the official documentation, rather than being in the repo - which we have to maintain.

Reply:
documentation has been updated to reference artifactory docs